test/helpers/utils.go

```diff
@@ -671,3 +671,20 @@ func DualStackSupported() bool {
 	// We only have DualStack enabled in Vagrant test env.
 	return GetCurrentIntegration() == ""
 }
+
+// DualStackSupportBeta returns true if the environment has a Kubernetes version that
+// has support for k8s DualStack beta API types.
+func DualStackSupportBeta() bool {
+	k8sVersion := GetCurrentK8SEnv()
+	switch k8sVersion {
+	// DualStack support was promoted to beta with API types finalized in k8s 1.21
+	// The API types for dualstack services are same in k8s 1.20 and 1.21 so we include
+	// K8s version 1.20 as well.
+	// https://github.com/kubernetes/kubernetes/pull/98969
+	case "1.12", "1.13", "1.14", "1.15", "1.16", "1.17", "1.18", "1.19":
```

Comment:
I thought I recall seeing some existing code where we can do version "arithmetic" like `<1.20`? Anyway this is pretty minor, up to you